gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -23,18 +23,14 @@
                 "scope": {
                   "type": "string",
                   "description": "Scope of the key",
-                  "default": "global",
+                  "default": "service",
```

Comment:
I think this change of behavior should be documented in the changelog.

Reply:
This policy was included in a 'release candidate' version that can become final. If it does, this change will be included in the next release and be considered a breaking change. I think we should be clear about that in the changelog.

Reply:
@davidor I'm not sure if it is a breaking change. Just changing the default value will affect only new instances of this policy.

Reply:
You're right @mikz :+1: 
This is not a breaking change, sorry for the confusion.